doc/source/cluster/metrics.md

```diff
@@ -178,6 +178,11 @@ To visualize and monitor collected metrics, there are 3 common paths:
 2. **Recommended**: Use Ray Dashboard which embeds Grafana visualizations and look at metrics together with logs, Job info and so on in a single pane of glass.
 3. **Manual**: Set up Grafana or other tools like CloudWatch, Cloud Monitoring, and Datadog from scratch.
 
+```{admonition} Note
```

Comment:
Can we add this as a troubleshooting tip and include this tip in both Prometheus and Grafana troubleshooting sections?

Something like 

>  **Troubleshooting: Loading Ray configurations with Docker Compose**
> In the Ray container, the symbolic link "/tmp/ray/session_latest/metrics" points to the latest active Ray session. However, Docker does not support the mounting of symbolic links on shared volumes and you may fail to load the Prometheus and Grafana configuration files and default dashboards.
>
>To fix this issue, employ an automated shell script for seamlessly transferring the necessary Grafana and Prometheus configurations and dashboards from the Ray container to a shared volume. To ensure a proper setup, the shared volume should be mounted on the respective paths for the containers, which contains the recommended configurations and default dashboards to initiate the Prometheus and Grafana servers.

Reply:
Sure